bower.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "iron-elements",
+  "version": "0.0.0",
+  "description": "Iron elements are a set of visual and non-visual utility elements. They include elements for working with layout, user input, selection, and scaffolding apps.",
+  "keywords": [
+    "web-components",
+    "polymer",
+    "utility",
+    "user-input",
+    "selection"
+  ],
+
+  "main": "iron-elements.html",
+  "dependencies": {
+    "iron-ajax":        "PolymerElements/iron-ajax#master",
+    "iron-selector":    "PolymerElements/iron-selector#master",
+    "iron-icon":        "PolymerElements/iron-icon#master",
+    "iron-icons":       "PolymerElements/iron-icon#master",
+    "iron-iconset":     "PolymerElements/iron-iconset#master",
+    "iron-iconset-svg": "PolymerElements/iron-iconset-svg#master",
+    "iron-image":       "PolymerElements/iron-image#master",
+    "iron-doc-viewer":  "PolymerElements/iron-doc-viewer#master",
+    "iron-meta":        "PolymerElements/iron-meta#master"
```

Comment:
We are no longer using branches to refer to dependencies. Please change the dependency version to `#^0.8.0` for all of these.